docs/index.md

```diff
@@ -28,22 +28,20 @@ F´ has the following features:
 
 ### Discontinued:
 - [Community Forum and Mailing List](https://groups.google.com/d/forum/fprime-community) 
-    - **Note**: Community forum has been replaced by [Discussions](https://github.com/nasa/fprime/discussions) since mid june 2021.
+    - **Note**: Community forum has been replaced by [Discussions](https://github.com/nasa/fprime/discussions) since mid June 2021.
 
 ## F´ Documentation
 
 ### F´ Release
 - [Latest Documentation](./latest.md)
 - [v3.1.0 Documentation](https://nasa.github.io/fprime/v3.1.0)
 - [v3.0.0 Documentation](https://nasa.github.io/fprime/v3.0.0)
-- [v2.1.0 Documentation](https://nasa.github.io/fprime/v2.1.0)
-- [v2.0.1 Documentation](https://nasa.github.io/fprime/v2.0.1)
-- [v2.0.0 Documentation](https://nasa.github.io/fprime/v2.0.0)
+- [v2.0 Series Documentation](https://nasa.github.io/fprime/v2.0.0)
```

Comment:
This is specifically the 2.0.0 documentation, not the documentation for all of v2.x.x.  For now, we should keep the full set of links.

v.1.5 is different, as at the time we didn't version documentation and thus didn't have specific links.

Otherwise this PR looks good!

Reply:
Good to know, thank you - latest commit should rollback to full set of links!